knx.js

```diff
@@ -207,6 +207,40 @@ module.exports = function (RED) {
                     buf.writeFloatLE(value, 0);
                     value = buf;
                     break;
+                case '10':   //3Byte fron timestamp
```

Comment:
May i ask you to provide few test cases for both DPT10 and DPT11?
For example:
`1550745501013` -> `Buffer[ 141, 38, 21 ]`

I'll add unit tests on the base of Mocha + ChaiJS